samples/DotNet/Azure.Messaging.ServiceBus/TopicSubscriptionFilters/CreateTopicAndSubscriptions/CreateTopicsAndSubscriptions.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Suggest switching to .NET6.
.NET 5 End of Life is 2022-05-08

Reply:
Thanks @SeanFeldman.